app.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] OPTIONAL:  Some of these are either not needed or have a solution:

- `/* eslint-disable max-classes-per-file */` does not seem to be needed.
- `/* eslint-disable no-unused-vars */` also does not seem to be needed.
- `/* eslint-disable no-use-before-define */`  just move your line 34 to line 15.  Then, move `addBookToList` function before `displayBooks`.  

After deleting the first 2, and correcting the third one, the project still works (tested it).  You could give it a try! 💪

Reply:
Thank you very much. I really appreciate your effort in improving my code